config.go

```diff
@@ -215,10 +215,12 @@ type aniConfig struct {
 	StepDuration   float64     `json:"stepDuration"`
 	TurnToDuration float64     `json:"turnToDuration"`
 
-	AniType aniTypeEnum   `json:"anitype"`
-	OnStart *actionConfig `json:"onStart"` //start
-	OnPlay  *actionConfig `json:"onPlay"`  //play
-	IsLoop  bool          `json:"isLoop"`
+	AniType      aniTypeEnum   `json:"anitype"`
+	OnStart      *actionConfig `json:"onStart"` //start
+	OnPlay       *actionConfig `json:"onPlay"`  //play
+	IsLoop       bool          `json:"isLoop"`
+	IsKeepOnStop bool          `json:"isKeepOnStop"`
```

Comment:
It may be better to add description

> some animations do not automatically switch to the default animation after completion of playback.

here as comment for field `IsKeepOnStop`, so we can benefit from Go Doc Comments in future maintenance.

Reply:
ok